### PR TITLE
chore: add DefaultSelector method ut

### PR DESCRIPTION
**Pull request by googs1025**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
- increase DefaultSelector method ut coverage
#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
None
```

### CODE REVIEW

File: pkg/scheduler/framework/plugins/helper/spread_test.go [hunk 1]
```diff
@@ -99,3 +101,159 @@ func TestGetPodServices(t *testing.T) {
...
+		replicationLabelVal = "replicationLabelVal"
+
+		statefulSetLabelKey = "statefulSetLabelKey"
+		statefulSetLabelVal = "statefulSetLabelVal"
+	)
+
+	fakeInformerFactory := informers.NewSharedInformerFactory(&fake.Clientset{}, 0*time.Second)
+
+	// Create fake service
+	service := &v1.Service{
```
Comment (kerthcet) [inline pkg/scheduler/framework/plugins/helper/spread_test.go:130]: Why the initialization is different with other objects, what if service not exists.

File: pkg/scheduler/framework/plugins/helper/spread_test.go [hunk 2]
```diff
@@ -99,3 +101,159 @@ func TestGetPodServices(t *testing.T) {
...
+					MatchLabels: map[string]string{
+						statefulSetLabelKey: statefulSetLabelVal,
+					},
+				},
+			},
+		}
+		return fakeInformerFactory.Apps().V1().StatefulSets().Informer().GetStore().Add(statefulSet)
+	}
+
+	tests := []struct {
```
Comment (kerthcet) [inline pkg/scheduler/framework/plugins/helper/spread_test.go:198]: also need one more testcase to cover the default case.

File: pkg/scheduler/framework/plugins/helper/spread_test.go (hunk 1 above)
Comment (googs1025) [reply pkg/scheduler/framework/plugins/helper/spread_test.go]: I added  `addFakeService func() error` method and also injected when running test case

File: pkg/scheduler/framework/plugins/helper/spread_test.go (hunk 2 above)
Comment (googs1025) [reply pkg/scheduler/framework/plugins/helper/spread_test.go:198]: I added a default case
```go
{
	name: "DefaultSelector for default case",
	pod: st.MakePod().Name(podName).
		Namespace(namespace).Label(podLabelKey, podLabelVal).
		Obj(),
	expect:          labels.Set{},
	addFakeService:  nil,
	addFakeResource: nil,
},
```

File: pkg/scheduler/framework/plugins/helper/spread_test.go [hunk 3]
```diff
@@ -99,3 +101,180 @@ func TestGetPodServices(t *testing.T) {
...
+		name            string
+		pod             *v1.Pod
+		expect          labels.Set
+		addFakeService  func() error
```
Comment (kerthcet) [inline pkg/scheduler/framework/plugins/helper/spread_test.go:202]: Why we need this? Doesn't this can also be part of `addFakeResource`.

Comment (googs1025) [reply pkg/scheduler/framework/plugins/helper/spread_test.go]: I changed addFakeResourceList to slice, and passed the func of service and other resources respectively.
```go
tests := []struct {
		name                string
		pod                 *v1.Pod
		expect              labels.Set
		addFakeResourceList []func() error
	}
```

**Review by kerthcet — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**